oat-sa/extension-tao-proctoring
# [ACTP-314] Delivery/Proctoring Page - Refactor test card

Related to : https://oat-sa.atlassian.net/browse/ACTP-314
 
Refactor test card on proctoring authorization screen to make it accessibility friendly

#### How to test
 
- Prepare an instance of TAO with taoAct extension
- Prepare a delivery with required proctoring
- Try to execute delivery
- On proctoring authorization screen:
  1. check that actions buttons are keyboard accessible
  2. check focus styles of action buttons

 
  
#### Dependencies
   
Companion PR :
 - [ ] https://github.com/oat-sa/extension-tao-act/pull/1321
